From 18f3dc6df030f15a122777158d44707cbc830ebd Mon Sep 17 00:00:00 2001
From: Andrey Borzenkov <arvidjaar@gmail.com>
Date: Wed, 24 Jun 2015 09:25:57 +0100
Subject: [PATCH 902/907] chkconfig: Delete a service before adding it.

(cg) Not 100% sure of the need for this one but I guess it keeps things clean.
---
 chkconfig.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chkconfig.c b/chkconfig.c
index 6399ef1..d88ab5d 100644
--- a/chkconfig.c
+++ b/chkconfig.c
@@ -742,6 +742,7 @@ int main(int argc, const char ** argv) {
 	    usage(progname);
 
 	name = basename(name);
+	delService(name, type, -1);
 	r = addService(name, type);
         reloadSystemd();
 
-- 
2.3.2

